### PR TITLE
Add issue templates

### LINKED ISSUE #29: Create issue templates
The bug template here is particularly useful: https://github.com/bevyengine/bevy/pull/4652

I would also probably grab the Enhancement issue template from Bevy.

### PR DESCRIPTION
**Pull request by asur4s**
Fixes #29

- I copied over the bug template from https://github.com/bevyengine/bevy/pull/4652 
  - modify the label as `bug`
  - remove content related to bevy
  - Add question about platform. My description may not be clear, please let me know if I should make adjustments.
- I copied over the bug template from [bevy/feature_request.md](https://github.com/bevyengine/bevy/blob/main/.github/ISSUE_TEMPLATE/feature_request.md), just modify the label as `enhancement`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: ''
+labels: bug
+assignees: ''
+---
+
+## stretch2 version
+
+The release number or commit hash of the version you're using.
+
+## platform
```
Comment (TimJentzsch) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
## Platform
```
For consistency :)

**Review by TimJentzsch — APPROVED** [review]

**Comment by TimJentzsch** [comment]
@asur4s Can you change the reference to the issue in the PR description to `Fixes #29`?
Then GitHub will automatically close this issue when this PR is merged and show it on there as well :)

Maybe we should add that to the CONTRIBUTING.md for future reference.

**Comment by asur4s** [comment]
> @asur4s Can you change the reference to the issue in the PR description to `Fixes #29`? Then GitHub will automatically close this issue when this PR is merged and show it on there as well :)
> 
> Maybe we should add that to the CONTRIBUTING.md for future reference.

Actually, I saw it in the CONTRIBUTING.md, just not sure if I have to add `fixes`. I'm a newbie.

**Comment by TimJentzsch** [comment]
No worries! It's a special keyword in GitHub: <https://github.blog/2013-01-22-closing-issues-via-commit-messages/>
It also shows up on the issue page like this:
GitHub showing on the issue page that it will be fixed by this PR

You can use the keywords `close`, `closes`, `closed`, `fixes` and `fixed` in front of the issue number for this, they all work the same :)

**Comment by asur4s** [comment]
> No worries! It's a special keyword in GitHub: https://github.blog/2013-01-22-closing-issues-via-commit-messages/ It also shows up on the issue page like this: GitHub showing on the issue page that it will be fixed by this PR
> 
> You can use the keywords `close`, `closes`, `closed`, `fixes` and `fixed` in front of the issue number for this, they all work the same :)

Good job, get it!

**Comment by alice-i-cecile** [comment]
Awesome, thank you :)

**Comment by alice-i-cecile** [comment]
@asur4s this is ready to merge as soon as you fix the Markdown lint errors in CI :)

**Comment by asur4s** [comment]
Just now, I checked with markdownlint-cli and passed, the CI check should be able to pass.

**Comment by alice-i-cecile** [comment]
Awesome, thanks a ton. Merging now.

**Pull request merged**